packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -106,6 +107,7 @@ function getTransformerFactory(
     if (options?.addSearchableTransformer) {
       transformerList.push(new SearchableModelTransformer());
     }
+    transformerList.push(new MapsToTransformer()); // this transformer must execute at the end
```

Comment:
Why does this need to be last?

Reply:
leftover from a previous implementation I was experimenting with. It can go in the main list of transformers now